### PR TITLE
Bug#1543291 - Added a note for Azure disk delays

### PR DESCRIPTION
**Pull request by gaurav-nelson**
For https://bugzilla.redhat.com/show_bug.cgi?id=1543291

### CODE REVIEW

**Comment by gaurav-nelson** [comment]
@openshift/team-documentation size/S PR PTAL 👀 

QE review is pending but I am not expecting any major changes.

**Comment by rootfs** [comment]
looks good, thank you!

File: install_config/persistent_storage/persistent_storage_azure.adoc [hunk 1]
```diff
@@ -131,6 +131,16 @@ xref:../../dev_guide/persistent_volumes.adoc#dev-guide-persistent-volumes[reques
 storage using persistent volume claims], which can now use your new persistent
 volume.
 
+[IMPORTANT]
+====
+For a pod that has a mounted volume through an Azure disk PVC, scheduling the
+pod  to a new node takes few minutes. Wait for two to three minutes to complete
```
Comment (ahardin-rh) [inline install_config/persistent_storage/persistent_storage_azure.adoc]: extra space between "pod" and "to"

s/takes few minutes/takes a few minutes

extra space after "creation"

**Pull request merged**